vendor/bat-native-ledger/src/bat/ledger/internal/media/reddit.cc

```diff
@@ -194,9 +194,10 @@ std::string MediaReddit::GetUserId(const std::string& response) {
   if (response.empty()) {
     return std::string();
   }
-
-  std::string id(braveledger_media::ExtractData(
-      response, "hideFromRobots\":false,\"id\":\"t2_", "\""));
+  const std::string pattern = ExtractData(response, "hideFromRobots\":",
```

Comment:
nit: `we always use braveledger_media:: ExtractData`

Reply:
Fixed